File: 20242/Clase 30_09/ComposicionAgregracion/Clase/Ejecutar.java
public class Ejecutar {
    public static void main(String[] args) {
        
        Cliente objCliente1 = new Cliente(123456789, "Andres", "Lopez");

        Cuenta objCuenta1 = new Cuenta("709876","Ahorros",0.0);

        Banco objBanco1 = new Banco("09922344", "Banco UCC", objCuenta1);

        System.out.println(objBanco1);
        System.out.println(objBanco1.esCliente(objCliente1));
    }
}

File: JAVASCRIPT/ComposicionAgregacion/Clase 30_09/JS/Ejecutar.js
import Banco from "./Banco.js";
import Cliente from "./Cliente.js"
import Cuenta from "./Cuenta.js"

class Ejecutar {
    constructor() {
        
        const objCliente1 = new Cliente(123443, "Andres", "Lopez");
        const objCuenta1 = new Cuenta("1234","Ahorros",0.0);
        const objBanco1 = new Banco("12223","Banco UCC",objCuenta1);

        console.log(objBanco1.toString());
        console.log(objBanco1.esCliente(objCliente1));
        
        
        //console.log(objCuenta1.consignar(objCuenta1.getNroCuenta(), 140000.0))
        console.log(objCuenta1.consignarMsg(objCuenta1.getNroCuenta(), 120000.0));
        console.log(objCuenta1.toString());

        console.log(objBanco1.consignarDinero(12200));
        
        
    }
}

new Ejecutar();